src/meteors/visualize/shap_visualize.py

```diff
@@ -0,0 +1,696 @@
+        feature_values = feature_values[observation_index]
+    if target is not None:
+        explanations_values = explanations_values[..., target]
+
+    fig = shap.plots.force(
+        base_value=explainer._explainer.expected_value[observation_index],
+        shap_values=explanations_values,
+        features=feature_values,
+        feature_names=explanation.feature_names,
+        matplotlib=True,
+        show=use_pyplot,
+        **kwargs,
+    )
+    return fig
+
+
+def beeswarm(
+    explainer: HyperSHAP,
+    explanation: SHAPExplanation,
+    target: int | None = None,
+    observation_index: int | None = None,
+    use_pyplot: bool = False,
+    ax: Axes | None = None,
+) -> Axes | None:
+    """Create a beeswarm plot for the given explanation. A reference for the plot might be found here: https://shap.readthedocs.io/en/latest/generated/shap.plots.beeswarm.html
+
+
+    Args:
+        explainer (HyperSHAP): A HyperSHAP explainer object used to generate the explanation.
+        explanation (SHAPExplanation): A SHAPExplanation coming from the meteors package. This object contains the explanations for the model. It can contain both the local and global explanations.
+        target (int | None, optional): In case the explained model outputs multiple values, this field specifies which of the outputs we want to explain. Defaults to None.
+        observation_index (int | None, optional): An index of observation that should be locally explained. This plot supports also global explanations, so this value is not necessary and can be set to None.
+        In case the passed explanation object is already local and contains data about only a single observation, this value could be also set to None. Defaults to None.
+        use_pyplot (bool, optional): If True, uses pyplot to display the image and shows it to the user. If False, returns the figure and axes objects. Defaults to False.
+        ax (matplotlib.axes.Axes, optional): If provided, the plot will be displayed on the passed axes.
+
+    Returns:
+        Axes | None:
+            If use_pyplot is False, returns the axes object.
+            If use_pyplot is True, returns None.
+    """
+    validate_explanations_and_explainer_type(explainer, explanation)
+    target = validate_target(target, explanation, require_single_target=True, plot_type="beeswarm")
+    observation_index = validate_observation_index(observation_index, explanation, require_local_explanation=False)
+
+    explanations = explanation.explanations
+    if target is not None:
+        explanations = explanations[..., target]
+    if observation_index is not None or explanation.is_local_explanation:
+        raise ValueError("The beeswarm plot does not support local explanations.")
+
+    # fig = shap.plots.beeswarm(explanations, ax=ax, show=use_pyplot)
+    # Current release of SHAP does not support passing ax parameter to the beeswarm plot, even though it is present in the documentation.
```

Comment:
Remove what is not needed
```suggestion
    # Current release of SHAP does not support passing ax parameter to the beeswarm plot, even though it is present in the documentation.
```